modules/RTC/TPCUtils.js

```diff
@@ -79,12 +79,19 @@ export class TPCUtils {
     _findTransceiver(mediaType, localTrack = null) {
         let transceiver = null;
 
-        if (localTrack) {
-            transceiver = this.pc.peerconnection.getTransceivers()
-                .find(t => t.sender?.track?.id === localTrack.getTrackId());
-        } else if (mediaType) {
+        // Check if the local track has been removed from the peerconnection already.
+        const trackRemoved = !localTrack
+            || (localTrack
+                && browser.doesVideoMuteByStreamRemove()
```

Comment:
we're using this combination of conditions so often that it feels would make sense to add a helper method to jitsi track "was removed for video mute?". Just a suggestion doesn't need to happen in this PR. I just think it would be easier to read one condition rather than having to connect 3 every time.

Reply:
Yeah makes sense Pawel! Will add it in the next one.